src/pages/calculator/Calculator.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import CalculatorApp from '../../components/calculator/Calculator';
+
+const Calculator = () => (
+  <section className="welcome container">
+    <h1>Do Some Math!</h1>
+    <div>
+      <CalculatorApp />
+    </div>
+  </section>
```

Comment:
- [ ] The general layout should be the same as the given design. The layout for the `calculator` component is not looking the same. In a real project as well you would have given some design and you have to make a similar one that's why we encourage students to do the same thing here.

### Your design 
![Screenshot from 2021-09-30 01-52-34](https://user-images.githubusercontent.com/22434660/135343306-896d2349-814d-4196-a9fb-eef097e76764.png)

### Given design

![Screenshot from 2021-09-30 01-56-24](https://user-images.githubusercontent.com/22434660/135343385-ade349ce-8e4b-4155-9694-2f57b64058e7.png)